mkdocs/docs/HPC/setting_up_python_virtual_environments.md

```diff
@@ -0,0 +1,221 @@
+# Python Virtual Environments (venv's)
+
+## Introduction
+
+Python virtual environments (venv's) are a way to create isolated Python environments.
+You can install packages in a venv without affecting the system Python installation.
+Because a normal user cannot install packages globally, 
+venv's are a way to install packages without needing root access.
+This is useful when you need to use a package that is not available as a module on the HPC cluster.
+
+## Creating a venv
+
+A venv can be created with the following commands:
```

Comment:
```suggestion
A Python virtual environment can be created with the following commands:
```

Reply:
fixed in [33ac8a2](https://github.com/hpcugent/vsc_user_docs/pull/639/commits/33ac8a20833fe1dcca543de078d03d72a4a89e7f)